microcosm_pubsub/consumer.py

```diff
@@ -95,6 +95,9 @@ def configure_sqs_consumer(graph):
     if graph.metadata.testing:
         from mock import MagicMock
         sqs_client = MagicMock()
+    elif sqs_queue_url == "test":
```

Comment:
I needed this to test standby behavior with a real consumer daemon and not spew errors.

It would be slightly better to use something other than a `MagicMock` since we may not have that imported outside of local development.